js/components/report/iframe-answer-report-item-attachment.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { PureComponent } from "react";
+import { Map } from "immutable";
+import { handleGetAttachmentUrl, IAttachmentUrlRequest, IReportItemAnswerItemAttachment } from "@concord-consortium/interactive-api-host";
+import { IframeAnswerReportItemAttachmentAudio } from "./iframe-answer-report-item-attachment-audio";
+
+interface IProps {
+  answer: Map<any, any>;
+  item: IReportItemAnswerItemAttachment;
+  request?: any;
+}
+
+interface IState {
+  contentType?: string;
+  error?: string;
+  loading: boolean;
+  url?: string;
+}
+
+export class IframeAnswerReportItemAttachment extends PureComponent<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      loading: true
+    };
+    this.handleLoadAttachment = this.handleLoadAttachment.bind(this);
+  }
+
+  UNSAFE_componentWillMount() {
+    const { item, answer } = this.props;
+    const answerMeta = answer.toJS();
+    const attachment = answerMeta.attachments[item.name];
+    // TODO: Save contentType to Firebase when saving file in runtime? It's not currently
```

Comment:
Right now the only thing available in the attachment is `folder` and `publicPath`. There is no `contentType` value saved there or anywhere else in the answer. Would the best thing to do be to save `contentType` as a property of an attachment when the attachment is saved in runtime?